doc/news/2017-03-18_lcdproc.md

```diff
@@ -0,0 +1,274 @@
+# Elektrify LCDproc
+
+- guid: d52657b5-60da-4a21-9679-f8aacf6d6b72
+- author: Markus Raab
+- pubDate: Sat, 18 Mar 2017 20:43:03 +0100
+- shortDesc: Elektrify LCDproc
+
+
+> LCDproc is a piece of open source software that displays real-time
+> system information from your Linux/*BSD box on a LCD.
+
+LCDproc's [website](http://lcdproc.omnipotent.net/) is a bit outdated
+but LCDproc itself is now well-maintained by Harald Geyer on
+[GitHub](https://github.com/lcdproc/lcdproc) and had a
```

Comment:
I know it's a lost cause, but I try to stay out of the maintainer role as much as possible. So I'd prefer if I wasn't named as a maintainer or in any leading role.

Otherwise the text looks fine to me.